### ISSUE TITLE
[508] Accessibility violation on /node/add/landing_page

### ISSUE THREAD

**Issue by acrollet** [department-of-veterans-affairs/va.gov-cms#4022]
**Describe the defect**

cypress-axe reports:

```
 ┃ │    0    │ 'color-contrast' │ 'serious' │ 'Ensures the contrast between foreground and background colors             ┃ meets WCAG 2 AA contrast ratio thresholds' │ [ 'div:nth-child(2) > a[href$="release-notes"]' ] │   1   │
```

**To Reproduce**

Run accessibility tests.

**Expected behavior**

All accessibility tests should pass.
